configure.ac

```diff
@@ -37,6 +37,12 @@ AC_CONFIG_HEADERS([config.h])
 AC_DEFINE([CONFIG_H_INCLUDED], [], [defined if config.h included])
 PKG_PROG_PKG_CONFIG
 
+# set configure arguments
+AC_CONFIGURE_ARGS=`echo "${ac_configure_args}" | sed "s/ //"`
+# | sed "s/'/\\\'/g"`
```

Comment:
Quoting of variables is just learned best practice. The intention is that the args can be used directly without any mangling to allow using it with scripts.